com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/Logic/AllNode.cs

```diff
@@ -19,7 +19,7 @@ internal class AllNode : IStandardNode
         public static NodeUIDescriptor NodeUIDescriptor => new(
             Version,
             Name,
-            tooltip: "returns true if all components of the input In are non-zero",
+            tooltip: "Checks if all components of the input *In* are non-zero values",
```

Comment:
```suggestion
            tooltip: "Checks if all components of the input In are non-zero values",
```

We didn't keep this up elsewhere, so unless we're bolding all port names, let's just leave this for now.